recipes/python/retail/retail/datasaver.py

```diff
@@ -21,12 +21,8 @@ def save(configProperties, prediction):
     print("Datasaver Start")
     print("Setting up Writer")
 
-    catalog_url = "https://platform.adobe.io/data/foundation/catalog"
-    ingestion_url = "https://platform.adobe.io/data/foundation/import"
-
-    writer = DataSetWriter(catalog_url=catalog_url,
-                           ingestion_url=ingestion_url,
-                           client_id=configProperties['ML_FRAMEWORK_IMS_USER_CLIENT_ID'],
+   
+    writer = DataSetWriter(client_id=configProperties['ML_FRAMEWORK_IMS_USER_CLIENT_ID'],
                            user_token=configProperties['ML_FRAMEWORK_IMS_TOKEN'],
                            service_token=configProperties['ML_FRAMEWORK_IMS_ML_TOKEN'])
```

Comment:
If you have time, can you run `pylint` if you haven't already?  We are standardizing on this tool for all python code.  Here is my output, (not all things need fixing)
```
No config file found, using default configuration
************* Module datasaver
C: 24, 0: Trailing whitespace (trailing-whitespace)
C: 38, 0: Exactly one space required after comma
        },  index=prediction.index))
         ^ (bad-whitespace)
C:  1, 0: Missing module docstring (missing-docstring)
E: 17, 0: Unable to import 'data_access_sdk_python.writer' (import-error)
C: 20, 0: Invalid argument name "configProperties" (invalid-name)
C: 20, 0: Missing function docstring (missing-docstring)
C: 31, 4: Invalid variable name "tenantId" (invalid-name)
```